devices/Chs6540/README.md

```diff
@@ -0,0 +1,73 @@
+# Chs6540 - Touch screen controller
+
+The Chs6540 is the touch screens controllers of the M5Stack [Tough](https://docs.m5stack.com/en/core/tough).
+
+## Documentation
+
+- [More information](https://github.com/m5stack/M5Tough/blob/master/src/M5Touch.cpp) on the touch controller, unfortunately, there is datasheet available.
```

Comment:
I believe there isn't a datasheet publicly available, correct? So this should should be

```suggestion
- [More information](https://github.com/m5stack/M5Tough/blob/master/src/M5Touch.cpp) on the touch controller, unfortunately, the datasheet isn't available.
```

Reply:
correct, there is none. So missed the not here :-D